tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -294,7 +303,7 @@ def _convert_ids_to_indices(self, ids: tuple):
 
         return ids
 
-    def convert(self, events):
+    def convert(self, events: np.ndarray) -> dict:
```

Comment:
> This is also known as a dataset

...suggests that we need to create some object for such a dataset, not sticking with a `dict`. Maybe make an issue for this? Might be addressed by [#23](https://github.com/ComPWA/tensorwaves/issues/23) though.

Reply:
Hmm, yeah I think we can discuss this in #23 as its related to this.